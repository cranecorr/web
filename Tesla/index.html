<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title>Document</title>
  <link rel="stylesheet" href="css/reset.css">
  <link rel="shortcut icon" href="images/favicon.png" type="image/x-icon">
  <!-- <link href="https://fonts.googleapis.com/css?family=Roboto|Roboto+Condensed:400,700&display=swap" rel="stylesheet"> -->
  <link href="https://fonts.googleapis.com/css?family=Roboto|Roboto+Condensed:400,700&display=swap&subset=cyrillic"
    rel="stylesheet">
  <link rel="stylesheet" href="css/slick.css">
  <link rel="stylesheet" href="css/style.css">
</head>

<body>
  <header class="header">
    <div class="container">
      <div class="header-top">
        <div class="logo">
          <a href="#">
            <img src="images/logo.png" alt="">
          </a>
        </div>
        <div class="header-btn">
          <div></div>
          <div></div>
          <div></div>
        </div>
      </div>
      <div class="header-content">
        <h1 class="header-content__title">
          Roadster
        </h1>
        <div class="header-content__text">
          Самый быстрый автомобиль в мире, с рекордным ускорением, дальностью и производительностью.
        </div>
        <a class="header-content__btn" href="#">Хочу тест драйв</a>
      </div>
    </div>
  </header>

  <section class="slider-top">
    <div class="slider">
      <div class="slider__item" style="background-image: url(images/slider-1.jpg);"></div>
      <div class="slider__item" style="background-image: url(images/slider-2.jpg);"></div>
      <div class="slider__item" style="background-image: url(images/slider-3.jpg);"></div>
    </div>
  </section>

  <section class="statistics">
    <div class="container">
      <div class="statistic-items">
        <div class="statistics-item">
          <div class="statisitics-items__title">
            Двигатель
          </div>
          <div class="statistics-item__num">
            2,1 <span>сек</span>
          </div>
          <div class="statistics-item__desc">
            разгон до 100 км
          </div>
        </div>
        <div class="statistics-item">
          <div class="statisitics-items__title">
            Батарея
          </div>
          <div class="statistics-item__num">
            1000 <span>км</span>
          </div>
          <div class="statistics-item__desc">
            запас хода
          </div>
        </div>
        <div class="statistics-item">
          <div class="statisitics-items__title">
            Скорость
          </div>
          <div class="statistics-item__num plus">
            400 <span>км/ч</span>
          </div>
          <div class="statistics-item__desc">
            макс скорость
          </div>
        </div>
      </div>
      <img src="images/pic-1.jpg" alt="" class="statistics-img">
    </div>
  </section>

  <section class="description">
    <div class="container">
      <div class="description-inner">
        <div class="description-inner__row">
          <div class="description__items">
            <div class="description__title">
              Скорость
            </div>
            <div class="description__text">
              Самый
              быстроразгоняющийся
              серийный автомобиль
              из когда-либо созданных
            </div>
          </div>
          <img src="images/pic-2.jpg" alt="" class="description__img">
        </div>
        <div class="description-inner__row">
          <img src="images/pic-3.jpg" alt="" class="description__img">
          <div class="description__items">
            <div class="description__text">
              <ul>
                <li>Датчик дождя</li>
                <li>Датчик света</li>
                <li>Автопилот</li>
                <li>Навигационная система</li>
                <li>Обогрев зеркал</li>
                <li>Усилитель руля</li>
                <li>Панорамная крыша</li>
                <li>Ксеноновые фары</li>
              </ul>
            </div>
            <div class="description__title right">
              Преимущества
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>





  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
  <script src="js/slick.min.js"></script>
  <script src="js/jquery.mousewheel.min.js"></script>
  <script src="js/main.js"></script>
</body>

</html>