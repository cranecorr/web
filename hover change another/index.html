<div class="out">
<div class="in in1">

</div>
<div class="in in2">

</div>
</div>