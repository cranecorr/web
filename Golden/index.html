<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title>Document</title>
  <link rel="stylesheet" href="css/reset.css">
  <link rel="stylesheet" href="css/style.css">
</head>

<body>
  <header class="header">
    <div class="container">
      <div class="header__inner">
        <div class="logo">
          <img src="images/logo.png" alt="">
        </div>
        <nav class="menu">
          <ul>
            <li><a href="#">Home</a></li>
            <li><a href="#">Services</a></li>
            <li><a href="#">Portfolio</a></li>
            <li><a href="#">About</a></li>
            <li><a href="#">Contact</a></li>
          </ul>
        </nav>
        <div class="header__text">
          Welcome To Our Studio! <br>
          <span>It’s nice to Meet you</span>
          <div class="header__btn">
            <a href="#">Tell Me More</a>
          </div>
        </div>
      </div>
    </div>
  </header>

  <section class="service">
    <div class="container">
      <div class="service__title title">
        <h2>
          Services
        </h2>
      </div>
      <div class="service__text">
        Proin iaculis purus consequat sem cure.
      </div>
      <div class="service__items">
        <div class="service__item">
          <div class="service__item-icon">
            <img src="images/icon1.png" alt="">
          </div>
          <div class="service__item-title">
            E-Commerce
          </div>
          <div class="service__item-text">
            Proin iaculis purus consequat sem cure
            digni ssim. Donec porttitora entum suscipit
            aenean rhoncus posuere odio in tincidunt.
          </div>
        </div>
        <div class="service__item">
          <div class="service__item-icon">
            <img src="images/icon2.png" alt="">
          </div>
          <div class="service__item-title">
            Responsive Web
          </div>
          <div class="service__item-text">
            Proin iaculis purus consequat sem cure
            digni ssim. Donec porttitora entum suscipit
            aenean rhoncus posuere odio in tincidunt.
          </div>
        </div>
        <div class="service__item">
          <div class="service__item-icon">
            <img src="images/icon3.png" alt="">
          </div>
          <div class="service__item-title">
            Web Security
          </div>
          <div class="service__item-text">
            Proin iaculis purus consequat sem cure
            digni ssim. Donec porttitora entum suscipit
            aenean rhoncus posuere odio in tincidunt.
          </div>
        </div>
      </div>
    </div>
  </section>

  <section class="portfolio">
    <div class="container">
      <div class="portfolio__title title">
        <h2>
          Our portfolio
        </h2>
      </div>
      <div class="portfolio__text">
        <p>
          Proin iaculis purus consequat sem cure.
        </p>
      </div>
      <div class="portfolio__items">
        <div class="portfolio__item">
          <a href="#">
            <img src="images/portfolio-1.png" alt="">
          </a>
        </div>
        <div class="portfolio__item">
          <a href="#">
            <img src="images/portfolio-2.png" alt="">
          </a>
        </div>
        <div class="portfolio__item">
          <a href="#">
            <img src="images/portfolio-3.png" alt="">
          </a>
        </div>
      </div>
    </div>
  </section>

  <section class="about">
    <div class="container">
      <h2 class="about__title title">
        About Us
      </h2>
      <p class="about__text">
        Proin iaculis purus consequat sem cure.
      </p>
      <div class="about__items">
        <div class="about__item">
          <div class="about__item-img">
            <img src="images/about-1.jpg" alt="">
          </div>
          <div class="about__item-content">
            <div class="about__item-title">
              <span>July 2010</span>
              <p>
                Our Humble Beginnings
              </p>
            </div>
            <div class="about__item-text">
              Proin iaculis purus consequat sem cure
              digni ssim. Donec porttitora entum suscipit
              aenean rhoncus posuere odio in tincidunt. Proin
              iaculis purus consequat sem cure digni
              ssim. Donec porttitora entum suscipit.
            </div>
          </div>
        </div>
        <div class="about__item">
          <div class="about__item-img">
            <img src="images/about-2.jpg" alt="">
          </div>
          <div class="about__item-content">
            <div class="about__item-title">
              <span>January 2011</span>
              <p>
                Facing Startup Battles
              </p>
  
            </div>
            <div class="about__item-text">
              Proin iaculis purus consequat sem cure
              digni ssim. Donec porttitora entum suscipit
              aenean rhoncus posuere odio in tincidunt. Proin
              iaculis purus consequat sem cure digni
              ssim. Donec porttitora entum suscipit.
            </div>
          </div>
        </div>
        <div class="about__item">
          <div class="about__item-img">
            <img src="images/about-3.jpg" alt="">
          </div>
          <div class="about__item-content">
            <div class="about__item-title">
              <span>December 2012</span>
              <p>
                Enter The Dark Days
              </p>
            </div>
            <div class="about__item-text">
              Proin iaculis purus consequat sem cure
              digni ssim. Donec porttitora entum suscipit
              aenean rhoncus posuere odio in tincidunt. Proin
              iaculis purus consequat sem cure digni
              ssim. Donec porttitora entum suscipit.
            </div>
          </div>
        </div>
        <div class="about__item">
          <div class="about__item-img">
            <img src="images/about-4.jpg" alt="">
          </div>
          <div class="about__item-content">
            <div class="about__item-title">
              <span>february 2014</span>
              <p>
                Our Triumph
              </p>
            </div>
            <div class="about__item-text">
              Proin iaculis purus consequat sem cure
              digni ssim. Donec porttitora entum suscipit
              aenean rhoncus posuere odio in tincidunt. Proin
              iaculis purus consequat sem cure digni
              ssim. Donec porttitora entum suscipit.
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>

  <footer class="footer">
    
      © Copyright 2014 FreebiesXpress.com
    
  </footer>

  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
  <script src="js/main.js"></script>
</body>

</html>